packages/now-cli/test/dev/integration.js

```diff
@@ -654,31 +654,36 @@ test(
   })
 );
 
-test(
+test.only(
```

Comment:
```suggestion
test(
```

Reply:
Is there a way to enable a lint rule for this so we don't accidently merge these types of errors 🤔

Reply:
@styfle Based on your idea, I opened https://github.com/zeit/now/pull/4229 to add linting for `.only` and `.skip`.